aosp_diff/preliminary/external/drm_hwcomposer/0002-Fix-issue-and-add-some-properties.patch

```diff
@@ -0,0 +1,290 @@
+From 0aac0bfcd243dc344c8e771a8eadd46e2b6adec8 Mon Sep 17 00:00:00 2001
+From: "Li, HaihongX" <haihongx.li@intel.com>
+Date: Thu, 26 Aug 2021 09:46:08 +0800
+Subject: [PATCH] Fix issue and add some properties
+
+1. Fix issue caused by crtc count and connector count doesn't match
+2. add property 'vendor.hwcomposer.connector.id' to set connector
```

Comment:
please add/explain more detail on purpose the properties added and how to set/use the properties